cypress/integration/homepage.spec.js

```diff
@@ -120,26 +122,6 @@ describe("Sparrow Application", () => {
       "contain",
       "Cypress Test Node"
     );
-    // Enter a second new node name
-    cy.get('[data-testid="form-input-node-name"]')
-      .clear()
-      .type("Other Node Name");
-    // Enter a second new node location
-    cy.get('[data-testid="form-input-node-location"]').clear().type("Garage");
-    //Click the Submit button
-    cy.get(".ant-form").submit();
-    // Verify the node name is now updated to "Other Node Name"
-    cy.get('[data-testid="node-name"]', { timeout: 50000 }).should(
-      "contain",
-      "Other Node Name"
-    );
-    //Click the sparrow Logo to return to the homepage
-    cy.get('[data-testid="logo"]').click({ force: true });
-    // verify the node location is now updated to "Garage"
-    cy.get('[data-testid="node-location"]', { timeout: 15000 }).should(
-      "contain",
-      "Garage"
-    );
```

Comment:
Yes - that's reminding me why I did that. I'll add it back and let our verification be confirming the name change on the node details page via the header instead of navigating back to the main dashboard view